pkg/kosmosctl/install/install.go

```diff
@@ -185,118 +209,146 @@ func (o *CommandInstallOptions) runClusterlink() error {
 		}
 	}
+	if err = util.WaitDeploymentReady(o.Client, clusterlinkDeployment, 120); err != nil {
+		return fmt.Errorf("kosmosctl install clusterlink run error, deployment options failed: %v", err)
+	} else {
+		klog.Info("Deployment clusterlink-network-manager has been created.")
+	}
 
 	return nil
 }
 
-func (o *CommandInstallOptions) runClusterrouter() error {
+func (o *CommandInstallOptions) runClustertree() error {
+	klog.Info("Start creating kosmos-clustertree...")
 	namespace := &corev1.Namespace{}
 	namespace.Name = o.Namespace
-	_, err := o.Client.CoreV1().Namespaces().Create(context.TODO(), namespace, metav1.CreateOptions{})
-	if err != nil {
-		if !apierrors.IsAlreadyExists(err) {
-			return fmt.Errorf("kosmosctl install clusterrouter run error, namespace options failed: %v", err)
+	_, err := o.Client.CoreV1().Namespaces().Get(context.TODO(), util.DefaultNamespace, metav1.GetOptions{})
+	if err != nil && apierrors.IsNotFound(err) {
+		_, err = o.Client.CoreV1().Namespaces().Create(context.TODO(), namespace, metav1.CreateOptions{})
+		if err != nil {
+			return fmt.Errorf("kosmosctl install clustertree run error, namespace options failed: %v", err)
 		}
+		klog.Info("Namespace kosmos-system has been created.")
+	} else {
+		klog.Info("Namespace kosmos-system already exists.")
 	}
 
-	hostKubeconfig, err := os.ReadFile(o.HostKubeConfig)
+	klog.Info("Start creating kosmos-clustertree ServiceAccount...")
+	clusterTreeServiceAccount, err := util.GenerateServiceAccount(manifest.ClusterTreeKnodeManagerServiceAccount, manifest.ServiceAccountReplace{
+		Namespace: o.Namespace,
+	})
 	if err != nil {
-		return fmt.Errorf("kosmosctl install clusterrouter run error, host-kubeconfig read failed: %v", err)
-	}
-	clusterRouterConfigMap := &corev1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "host-kubeconfig",
-			Namespace: o.Namespace,
-		},
-		Data: map[string]string{
-			"kubeconfig": string(hostKubeconfig),
-		},
+		return err
 	}
-	_, err = o.Client.CoreV1().ConfigMaps(o.Namespace).Create(context.TODO(), clusterRouterConfigMap, metav1.CreateOptions{})
+	_, err = o.Client.CoreV1().ServiceAccounts(o.Namespace).Create(context.TODO(), clusterTreeServiceAccount, metav1.CreateOptions{})
 	if err != nil {
 		if !apierrors.IsAlreadyExists(err) {
-			return fmt.Errorf("kosmosctl install clusterrouter run error, configmap options failed: %v", err)
+			return fmt.Errorf("kosmosctl install clustertree run error, serviceaccount options failed: %v", err)
 		}
 	}
+	klog.Info("ServiceAccount clustertree-knode-manager has been created.")
 
-	clusterRouterServiceAccount, err := util.GenerateServiceAccount(manifest.ClusterRouterKnodeServiceAccount, manifest.ServiceAccountReplace{
-		Namespace: o.Namespace,
-	})
+	klog.Info("Start creating kosmos-clustertree ClusterRole...")
+	clusterTreeClusterRole, err := util.GenerateClusterRole(manifest.ClusterTreeKnodeManagerClusterRole, nil)
```

Comment:
clusterTree should be clustertree, clustertree is a word

Reply:
ok, i'm done